src/reducers/appReducer.js

```diff
@@ -49,6 +49,9 @@ export default function appReducer(
         provider: '',
       },
     },
+    status: {
```

Comment:
Hmm... This is probably not needed? Or should I do something properly here and change this?

Reply:
I dont think it is needed. AFAIK `status` belongs to cluster state

Reply:
ACK. Removed.